material-ui/material-ui.scss

```diff
@@ -120,3 +120,106 @@ $swal2-close-button-hover-color: $swal2-close-button-color;
     }
   }
 }
+
+// Checkbox
+.swal2-checkbox {
+  z-index: 0;
+  position: relative;
+  > {
+    input {
+      appearance: none;
+      -moz-appearance: none;
+      -webkit-appearance: none;
```

Comment:
you can replace `-moz-appearance` and `-webkit-appearance` with a simple `appearance`. The SCSS compiler will translate those into browser specific styles.

Reply:
Great, didn't know about that will do.